src/main/java/stsjorbsmod/powers/PossessionPower.java

```diff
@@ -0,0 +1,63 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.JorbsMod;
+
+public class PossessionPower extends CustomJorbsModPower{
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(PossessionPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    private final AbstractMonster target;
+
+    public PossessionPower(AbstractMonster owner, int amount) {
+        super(STATIC);
+        this.type = PowerType.DEBUFF;
+        this.owner = owner;
+        this.amount = amount;
+        this.target = owner;
```

Comment:
I don't think the separate `target` field is necessary here; it feels misleading for a reader, since it's not actually the creature that's going to be targetted for the damage. You can just use `owner` in the `atStartOfTurn` implementation.